code/src/index.css

```diff
@@ -1,13 +1,139 @@
 body {
+  padding: 20px;
+  background-color: #242626;
+  display: flex;
+  justify-content: center;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+/* code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
+} */
+
+h1 {
+  font-family: Helvetica, sans-serif;
+  color: #fff;
+  font-size: 25px;
+  border-bottom: 2px solid rgba(255, 255, 255, 0.1);
+}
+
+a {
+  display: block;
+  font-family: Helvetica, sans-serif;
+  color: #a0a0a0;
+  font-size: 14px;
+  margin-bottom: 5px;
+  text-decoration: none;
+  /* border: 1px solid red; */
+}
+
+a:hover {
+  color: #fff;
+  text-decoration: underline;
+}
+
+.App {
```

Comment:
To not have the content spill over, it is a good idea to make an inner wrapper and specify a max width for the content. For example, all of your content is now inside the div that is "app" so if you specify a view width the content will not go passed those boundaries.